### PR TITLE
SRVCOM-1288 + SRVCOM-1664: Clean up attributes, prereqs

### PR DESCRIPTION
**Pull request by abrennan89**
Jiras: https://issues.redhat.com/browse/SRVCOM-1288 + https://issues.redhat.com/browse/SRVCOM-1664

Applies for OCP 4.6+

Direct preview links:
- https://deploy-preview-41944--osdocs.netlify.app/openshift-enterprise/latest/serverless/develop/serverless-listing-event-sources.html
- https://deploy-preview-41944--osdocs.netlify.app/openshift-enterprise/latest/serverless/functions/serverless-functions-annotations.html
- https://deploy-preview-41944--osdocs.netlify.app/openshift-enterprise/latest/serverless/functions/serverless-functions-accessing-secrets-configmaps.html
- https://deploy-preview-41944--osdocs.netlify.app/openshift-enterprise/latest/serverless/integrations/gpu-resources.html
- https://deploy-preview-41944--osdocs.netlify.app/openshift-enterprise/latest/serverless/cli_tools/installing-kn.html

### CODE REVIEW

File: _topic_maps/_topic_map.yml [hunk 1]
```diff
@@ -3231,6 +3231,8 @@ Topics:
     File: serverless-event-sinks
   - Name: Event delivery
     File: serverless-event-delivery
+  - Name: Listing event sources and event source types
+    File: serverless-listing-event-sources
```
Comment (matzew) [inline _topic_maps/_topic_map.yml:3235]: reviewed that part, and that does LGTM

**Review by lance — COMMENTED** [review]
/lgtm

**Comment by mgencur** [comment]
QE Approved

File: modules/serverless-installing-cli-web-console.adoc [hunk 2]
```diff
@@ -1,14 +1,22 @@
...
+.Prerequisites
 
-You can access the *Command Line Tools* page by clicking the image:../images/question-circle.png[title="Help"] icon in the top right corner of the web console and selecting *Command Line Tools* in the drop down menu.
+* You have logged in to the {product-title} web console.
+* The {ServerlessOperatorName} is installed on your {product-title} cluster. Once the {ServerlessOperatorName} is installed, you will see a link to download the Knative CLI (`kn`) for Linux (x86_64, amd64, s390x, ppc64le), macOS, or Windows from the *Command Line Tools* page in the {product-title} web console.
 
 .Procedure
 
-. Download the `kn` CLI from the *Command Line Tools* page.
+. Download the `kn` CLI from the *Command Line Tools* page. You can access the *Command Line Tools* page by clicking the image:../images/question-circle.png[title="Help"] icon in the top right corner of the web console and selecting *Command Line Tools* in the drop down menu.
```
Comment (adellape) [inline modules/serverless-installing-cli-web-console.adoc:18]: Consider "drop-down" per IBMSG.

File: serverless/functions/serverless-functions-annotations.adoc [hunk 3]
```diff
@@ -9,12 +9,10 @@ toc::[]
 
 You can add Kubernetes annotations to a deployed Serverless function by adding them to the `annotations` section in the `func.yaml` configuration file.
 
-[IMPORTANT]
-====
 There are two limitations of the function annotation feature:
 
-* Once a function annotation propagates to the corresponding Knative service on the cluster, it cannot be removed from the service by deleting it from the `func.yaml` file. You can remove the annotation from the Knative service by modifying the YAML file of the service directly, or by using the Developer Console.
+* Once a function annotation propagates to the corresponding Knative service on the cluster, it cannot be removed from the service by deleting it from the `func.yaml` file. You must remove the annotation from the Knative service by modifying the YAML file of the service directly, or by using the {product-title} web console.
```
Comment (adellape) [inline serverless/functions/serverless-functions-annotations.adoc:14]: s/Once/After/

File: modules/serverless-installing-cli-web-console.adoc [hunk 4]
```diff
@@ -1,14 +1,22 @@
...
 = Installing the Knative CLI using the {product-title} web console
 
-Once the {ServerlessOperatorName} is installed, you will see a link to download the Knative CLI (`kn`) for Linux (x86_64, amd64, s390x, ppc64le), macOS, or Windows from the *Command Line Tools* page in the {product-title} web console.
+You can use the following procedure to install the `kn` CLI by using the web console.
+
+.Prerequisites
 
-You can access the *Command Line Tools* page by clicking the image:../images/question-circle.png[title="Help"] icon in the top right corner of the web console and selecting *Command Line Tools* in the drop down menu.
+* You have logged in to the {product-title} web console.
+* The {ServerlessOperatorName} is installed on your {product-title} cluster. Once the {ServerlessOperatorName} is installed, you will see a link to download the Knative CLI (`kn`) for Linux (x86_64, amd64, s390x, ppc64le), macOS, or Windows from the *Command Line Tools* page in the {product-title} web console.
```
Comment (adellape) [inline modules/serverless-installing-cli-web-console.adoc:14]: s/Once/After/

**Pull request merged**